R/resp-body.R

```diff
@@ -115,9 +115,9 @@ check_content_type <- function(
   }
 
   abort(c(
-    glue("Unexpected content type '{content_type}"),
-    glue("Expecting {type}"),
-    if (!is.null(suffix)) paste0("Or suffix '", suffix, "'"),
+    glue("Unexpected content type '{content_type}'"),
+    glue("Expecting '{type}'"),
```

Comment:
The quotes are added above

Reply:
Yeah, I see that now that I ran / updated the snapshot tests.